routes/maps-id.js

```diff
@@ -4,11 +4,15 @@ const mapsQueries = require("../db/queries/maps");
 const pinsQueries = require("../db/queries/pins");
 
 router.get("/", (req, res) => {
-  let templateVar = { ApiKey: process.env.API_KEY };
+  const userId = req.cookies.user_id;
   mapsQueries
     .getMapData(req.params.id)
     .then((map) => {
-      templateVar.map = map[0];
+      templateVar = {
```

Comment:
the template var is not defined since you remove line 7 right?

Reply:
Thanks, I'll define it